website/docs/reference/snapshot-configs.md

```diff
@@ -206,98 +257,121 @@ snapshots:
 ## Configuring snapshots
 Snapshots can be configured in one of three ways:
 
-1. Using a `config` block within a snapshot
-2. Using a `config` [resource property](/reference/model-properties) in a `.yml` file
-3. From the `dbt_project.yml` file, under the `snapshots:` key. To apply a configuration to a snapshot, or directory of snapshots, define the resource path as nested dictionary keys.
+1. Defined in YAML files, typically in your snapshots directory (available in [Versionless](/docs/dbt-versions/versionless-cloud) or and dbt Core v1.9 and higher).
+2. Using a `config` block within a snapshot
+3. Using a `config` [resource property](/reference/model-properties) in a `.yml` file
```

Comment:
@dbeatty10 is the hierarchy correct here?

Reply:
The hierarchy looks correct to me! 👍

For the 1st item, you could link to the [snapshots directory](https://docs.getdbt.com/reference/project-configs/snapshot-paths) or [`snapshot-paths`](https://docs.getdbt.com/reference/project-configs/snapshot-paths).